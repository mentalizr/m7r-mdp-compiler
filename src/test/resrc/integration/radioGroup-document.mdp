@@name=AutoTest1
@@persistent

### Header 1

Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.

@radio-group[name="myRadioGroup" id="myRadioButton" "Situation" "Gedanken" "Gef√ºhl"]
    Drei Uhr am Nachmittag

Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium.