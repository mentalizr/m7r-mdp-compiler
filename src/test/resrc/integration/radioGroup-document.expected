<!--
@@name=AutoTest1
@@persistent
-->

<p class="h3 mt-4 mb-4">Header 1</p>
<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
<div id="myRadioButton" class="mb-2 ns_radiogroup">
    <div>Drei Uhr am Nachmittag</div>
    <div class="form-check form-check-inline">
        <input class="form-check-input" type="radio" name="myRadioGroup" id="myRadioButton_1" value="1">
        <label class="form-check-label" for="myRadioButton_1">Situation</label>
    </div>
    <div class="form-check form-check-inline">
        <input class="form-check-input" type="radio" name="myRadioGroup" id="myRadioButton_2" value="2">
        <label class="form-check-label" for="myRadioButton_2">Gedanken</label>
    </div>
    <div class="form-check form-check-inline">
        <input class="form-check-input" type="radio" name="myRadioGroup" id="myRadioButton_3" value="3">
        <label class="form-check-label" for="myRadioButton_3">Gef√ºhl</label>
    </div>
</div>
<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium.</p>